src/api/config/Config.ts

```diff
@@ -15,6 +15,7 @@ export default class Config implements StrykerOptions {
   timeoutFactor = 1.5;
   plugins: string[] = ['stryker-*'];
   port = 9234;
+  reporter = ['progress', 'clear-text'];
```

Comment:
When someone specifies some other reporter, will this array be overwritten or will the other reporter be added to this array?